public/modules/custom/grants_webform_print/src/Controller/GrantsWebformPrintController.php

```diff
@@ -153,33 +153,25 @@ private function fixWebformElement(array $element, string $key, array $translate
       return !str_contains($key, '#');
     });
 
-    // If there is some, then loop as long as there is som.
-    if ($children) {
+    // If there is some, then loop as long as there is some.
+    foreach ($children as $childKey) {
       $translations = $translatedFields[$key]['#element'] ?? $translatedFields;
-      foreach ($children as $childKey) {
-        $element[$childKey] = $this->fixWebformElement($element[$childKey], $childKey, $translations);
-      }
-
+      $element[$childKey] = $this->fixWebformElement($element[$childKey], $childKey, $translations);
     }
 
     // Apply translations to the element itself.
     if (!empty($translatedFields[$key])) {
       // Unset type since we do not want to override that from trans.
       unset($translatedFields[$key]['#type']);
-      if (isset($translatedFields[$key]['#description']) && !is_array($translatedFields[$key]['#description'])) {
-        $element['#description'] = $translatedFields[$key]['#description'] . '<br>';
-      }
-      else {
-        $element['#description'] = '';
-      }
-      if (isset($translatedFields[$key]['#help']) && !is_array($translatedFields[$key]['#help'])) {
-        $element['#description'] = $element['#description'] . $translatedFields[$key]['#help'];
-      }
+      $element['#description'] = preg_replace('/^(<br>)/',
+          '',
+          ($translatedFields[$key]['#attachment_desription'] ?? '') . '<br>' .
+          ($translatedFields[$key]['#description'] ?? '') . '<br>' .
+          ($translatedFields[$key]['#help'] ?? '')) . '<br>';
```

Comment:
Tämä on melko vaikea sulateltava yhdellä kertaa. Pilkkoisin hieman jotta luettavuus olisi parempi.

```suggestion
      $description = ($translatedFields[$key]['#attachment_desription'] ?? '') . '<br>' .
          ($translatedFields[$key]['#description'] ?? '') . '<br>' .
          ($translatedFields[$key]['#help'] ?? '')) . '<br>';
       // Remove leading line breaks.
      $element['#description'] = preg_replace('/^(<br>)*/', '', $description);
          
```

Huomaa lisätty tähti regexissä.